demosplan/DemosPlanCoreBundle/Logic/Statement/StatementDeleter.php

```diff
@@ -83,20 +84,27 @@ private function deleteOriginalStatement(Statement $originalStatement): void
         if (!$originalStatement->getChildren()->isEmpty()) {
             throw new InvalidArgumentException('Original-Statement to delete still has children.');
         }
-        $forReport = clone $originalStatement;
+        /** @var Connection $doctrineConnection */
+        $doctrineConnection = $this->getDoctrine()->getConnection();
+        $sql = 'DELETE FROM statement_import_email_original_statements WHERE original_statement_id = :id';
```

Comment:
why do you use dql here?

Reply:
New Call to an undefined method demosplan\DemosPlanCoreBundle\Repository\StatementRepository::getDoctrine().
Ans you still use dql. Is this needed?